beets/autotag/mb.py

```diff
@@ -285,6 +287,24 @@ def album_info(release):
     artist_name, artist_sort_name, artist_credit_name = \
         _flatten_artist_credit(release['artist-credit'])
 
+    ntracks = 0
+    for medium in release['medium-list']:
+        ntracks += len(medium['track-list'])
+
+    # for albums with more than 500 tracks
```

Comment:
A comment here to this effect might be nice:

> The MusicBrainz API omits [include details] when [condition]. So we use `browse_recordings` on chunks of tracks to recover the same information in this case.

Reply:
Done